TestKubernetes Failure in Vault 1.11

Vault 1.11 introduced the following errors in test_kubernetes.py. Creating an issue to track the resolution of this failure.

```
FAILED tests/integration_tests/api/auth_methods/test_kubernetes.py::TestKubernetes::test_configure_4_set_invalid_token_reviewer_jwt - AssertionError: '* not a compact JWS' not found in '1 error occurred:\n\t* square/go-jose: compact JWS format must have three parts\n\n, on post https://localhost:8200/v1/...
```

--- a/tests/integration_tests/api/auth_methods/test_kubernetes.py
+++ b/tests/integration_tests/api/auth_methods/test_kubernetes.py
@@ -55,7 +55,9 @@ class TestKubernetes(HvacIntegrationTestCase, TestCase):
                 token_reviewer_jwt="reviewer_jwt",
                 issuer="bob",
                 raises=exceptions.InternalServerError,
-                exception_message="* not a compact JWS",
+                exception_message="* not a compact JWS"
+                if utils.vault_version_lt("1.11.0")
+                else "compact JWS format must have three parts",
             ),
         ]
     )

